src/processorgen/mdx.tmpl

```diff
@@ -19,28 +21,52 @@ import Box from "@mui/system/Box";
 
 ## Configuration parameters
 
-{{ if eq (len .specification.parameters) 0 -}}
-  No configuration parameters.
-{{- else -}}
-  <table class="no-margin-table">
-    <tr>
-      <th>Name</th>
-      <th>Type</th>
-      <th>Default</th>
-      <th>Description</th>
-    </tr>
-    {{- range $name, $param := .specification.parameters }}
-    <tr>
-      <td>`{{ $name }}`</td>
-      <td>{{ $param.type }}</td>
-      <td>{{ formatParameterValue $param.default }}</td>
-      <td>
-{{ $param.description }}
-      </td>
-    </tr>
-    {{- end }}
-  </table>
-{{- end }}
+<Tabs groupId="config-params">
```

Comment:
WDYT about leaving just the YAML? It's very easy to use to look up parameters (which is what we'd do with the table)?

Reply:
If we want to only show yaml we should somehow also display the parameter type (for example as an inline comment).

Another way would be to flip this around and have YAML as the default and still allow the user to switch to the table view.

Reply:
At least to me, the first option is more appealing, because I can more easily work with the YAML. The 2nd option is good too.:)